_sass/uno.scss

```diff
@@ -1148,9 +1149,11 @@ hr {
 	color: #c7c7c7; }
 
 .post-meta__date {
-	margin-right: .5em; }
+	margin-right: 0;
+	text-transform: lowercase; }
```

Comment:
I don't think we should lowercase dates `Jun` vs `jun`

Reply:
All the tags are lowercase, stylistically the branding is lowercase so I would argue it looks out of place to be capitalised, but we're talking about a single, low-opacity character - does it matter?

Reply:
Is a proper noun, thought it looked amatuerish like we failed to write it properly. But yeah not a hill I'll die on if you strongly advocate it

"12 jun 2019" vs "12 Jun 2019"

Reply:
With everything else lowercase I think the negatives of people thinking we can't capitalise are removed, so ok to take the positive of stylistic consistency.